__unported__/l10n_it_DDT_webkit/__openerp__.py

```diff
@@ -31,6 +31,7 @@
     """,
     'author': 'Agile Business Group',
     'website': 'http://www.agilebg.com',
+    'license': 'GPL-3 or any later version',
```

Comment:
I setted the license refering to what is in header

    # it under the terms of the GNU General Public License as published by
    # the Free Software Foundation, either version 3 of the License, or
    # (at your option) any later version.